Formula/hotbuild.rb

```diff
@@ -0,0 +1,20 @@
+class Hotbuild < Formula
+  desc "Cross platfrom hot compilation tool for go"
+  homepage "https://hotbuild.ffactory.org"
+  url "https://github.com/wandercn/hotbuild/archive/refs/tags/v1.0.8.tar.gz"
+  sha256 "662fdc31ca85f5d00ba509edcb177b617d8d6d8894086197347cfdbd17dc7c2f"
+  license "MulanPSL-2.0"
+  head "https://github.com/lemnos/tt.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X version.Version=#{version}")
+  end
+
+  test do
+    output = "buildcmd = \"go build -o tmp/tmp_bin\""
```

Comment:
Can you add a version check?

Reply:
yeah, the version seems a bit off 😅

Reply:
added the build fix, should be good now.